lib/models/command.js

```diff
@@ -200,10 +202,12 @@ var Command = CoreObject.extend({
         return resolve('callHelp');
       }
 
-      this.analytics.track({
-        name:    'ember ',
-        message: this.name
-      });
+      if (isWhitelisted(this.name)) {
```

Comment:
Can we make this extensible so that our own internal addons with commands (fastboot, others) can opt in to tracking?